.gitignore

```diff
@@ -3,6 +3,7 @@
 .initializers
 docker-compose.override.yml
 *.pem
+.DS_Store
```

Comment:
Please add this to your [global gitignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/).

Reply:
Apologies.  Fair enough